lib/src/models/courseItem.dart

```diff
@@ -99,7 +99,7 @@ class CourseItem implements Comparable {
       courseSectionInformation: json["courseSectionInformation"] ?? null,
       classScheduleInformation: json["classScheduleInformation"] ?? null,
       sections: _getSections(json["sections"]),
-      categories: [json["genEd"]],
+      categories: List<String>.from(["genEd"]),
```

Comment:
```suggestion
      categories: List<String>.from(json["genEd"]),
```
Correct me if I'm wrong, but I think it would be like this?